packages/feature_flags/index.ts

```diff
@@ -13,11 +15,11 @@ export function checkFlag(flagName: FlagKey) {
 
    const enableFlag = 'enable-' + flagName;
 
+   const safeLocalStorage = useSafeLocalStorage();
+
    if (typeof flagValue === 'object') {
-      if (flagValue.userToggle && typeof localStorage !== 'undefined') {
-         if (localStorage?.getItem(enableFlag) === 'true') {
-            return true;
-         }
+      if (safeLocalStorage?.getItem(enableFlag) === 'true') {
```

Comment:
Are we guaranteed that `useSafeLocalStorage` returns not-`undefined` even on browsers without `localStorage` support?

Reply:
yes. its basically just an object that wraps calls to localstorage